demos/llama_chat/python/README.md

```diff
@@ -0,0 +1,77 @@
+# Llama 2 Chat  {#ovms_demo_llama_2_chat}
+
+### Introduction
+This demo showcases example usage of [Llama model](https://ai.meta.com/llama/) hosted via OpenVINO™ Model Server. The model used in this example can be found at [huggingface](https://huggingface.co/meta-llama/Llama-2-7b-hf) (~26GB). Steps below automate download and conversion steps to be able to load it using OpenVINO™. Example python script is provided to request answers to given question.
+
+### Download the model
+
+Prepare the environment:
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/demos/llama_chat/python
+virtualenv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+
+Download the `meta-llama/Llama-2-7b-hf` model from huggingface and save to disk in IR format using script below.  
+> NOTE: Download might take a while since the model is ~26GB.
+```bash
+python3 download_model.py
+```
+The model files should be available in models directory:
+```bash
+tree models
+
+models
+└── llama-2-7b-hf
+    └── 1
+        ├── config.json
+        ├── openvino_model.bin
+        └── openvino_model.xml
+```
+
+### Start OVMS with prepared Llama 2 model
+
+```bash
+docker run -d --rm -p 9000:9000 -v $(pwd)/models/llama-2-7b-hf:/model:ro openvino/model_server \
+    --port 9000 \
+    --model_name llama \
+    --model_path /model \
+    --plugin_config '{"PERFORMANCE_HINT":"LATENCY","NUM_STREAMS":1}'
+```
+
+### Run python client
+
+Run `client.py` script to run interactive demo. Available parameters:
+
+```bash
+python3 client.py -h
+
+usage: client.py [-h] --url URL --question QUESTION [--seed SEED] [--actor {general-knowledge,python-programmer}]
+
+Inference script for generating text with llama
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --url URL
+  --question QUESTION
+  --seed SEED
+  --actor {general-knowledge,python-programmer}
```

Comment:
Can we say more about those two arguments to better guide users?

Reply:
done